students/eowyn/session05/mailroom.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python
+
+import sys
+
+''' global variables'''
+donor_names = ["Margaret Atwood", "Fred Armisen",
+               "Heinz the Baron Krauss von Espy"]
+donations = [[300, 555], [240, 422, 1000], [1500, 2300]]
+DONORS = dict(zip(donor_names, donations))
+
+
+
+def safe_input():
+    return None
+
+def quit_code():
+    sys.exit()
+
+def return_to_menu():
+    ''' Raise a zero division error to trigger exit out of loop'''
+    return 5/0
```

Comment:
if you want to use Exception Handling for this, which is a fine idea, then you should specifically raise an Error:

raise ValueError

for instance, rather than using random code that happens to trigger an error.

As you mentioned, a custom Exception may the ideal, but you should be able to find one that makes sense.

try:

`print([ er for er in dir(__builtins__) if "Error" in er])`

To see all the built in errors. Maybe SystemError would work?